### PR TITLE
ch setting react router dom #181414555

### PR DESCRIPTION
**Pull request by NNesta**
## 1. Name of task
- Setup react router  (React-router-dom)
## 2. Pivot Tracker ID
-[ #181414555](https://www.pivotaltracker.com/story/show/181414555)
## 3. Branch Name
- `ch-setup-React-router-dom-##181414555`
## 4. Description of task

This task is about setting up in the project a route configuration in the barefoot nomad frontend using react route dom
## 5. Implementation
It was implemented by adding the react-router-dom package and routes/index.js file where there are initial routes and added also the view folder where there are some initial views and dummy views to separate them from the components.
## 6. How can this be manually tested
first of all, install all dependencies by using the command `yarn`
and do `yarn run dev`  to run  the server
go to routes:
- localhost:{port}/. ==> home page
- localhost:{port}/login ==> login page
- localhost:{port}/dummy ==> dummy page
## 7. test
run `yarn run test`

### CODE REVIEW

**Review by iyaremye-faustin — DISMISSED** [review]
Works fine

File: src/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const AllRoutes = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<HomeView />} />
+      <Route path="/login" element={<LoginView />} />
+      <Route path="/dummy" element={<App />} />
+      <Route path="*" element={<PageNotFoundView />} />
+    </Routes>
+  </Router>
```
Comment (rukundo-kevin) [inline src/routes/index.js:16]: All the routes needs to be encapsulated in `<BrowserRouter> `

**Review by rukundo-kevin — DISMISSED** [review]

File: src/routes/index.js (hunk 1 above)
Comment (NNesta) [reply src/routes/index.js:16]: Router is an alias of BrowserRouter
Check well on line 2 where I imported them

Comment (NNesta) [reply src/routes/index.js:16]: @rukundo-kevin

File: src/views/errors/PageNotFoundView.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (SergeNm) [inline src/views/errors/PageNotFoundView.js]: I think we shall structure ower folder like this:
- src/pages
- src/components

all components are imported in a certain page and the page is going to be used in routes

**Review by SergeNm — DISMISSED** [review]

**Review by mucyoalexaxel — DISMISSED** [review]

**Review by rukundo-kevin — DISMISSED** [review]

**Review by iyaremye-faustin — DISMISSED** [review]
Working fine

**Review by SergeNm — DISMISSED** [review]

**Review by iyaremye-faustin — APPROVED** [review]
Works fine

**Review by mucyoalexaxel — APPROVED** [review]
LGTM

**Review by EmyRukundo — APPROVED** [review]

**Pull request merged**